### PR TITLE
CCE-427 : Update help styling to be italic + updated help screen shot…

### PR DESCRIPTION
**Pull request by saadmhybris**
…s on patterns page.

### CODE REVIEW

File: src/less/components/_forms-validationandhelp.less [hunk 1]
```diff
@@ -46,9 +46,9 @@
 }
 
 .help-icon-inline:before{
-  margin: 0;
+  margin: 5px 0 0 0;
   font-size: 25px; 
-  color:#7e858d;
+  color:#686f88;
```
Comment (HybrisRawls) [inline src/less/components/_forms-validationandhelp.less]: this color needs to move to a variable

File: src/less/components/_forms-help.less [hunk 2]
```diff
@@ -36,18 +36,18 @@
...
    margin-top: 5px;
    margin-bottom: 10px;
-
    cursor: default;
+   font-style: italic;
+
    .hyicon{
       font-size: 25px;
       line-height: 0;
       color: currentColor;
```
Comment (HybrisRawls) [inline src/less/components/_forms-help.less]: it doesn't look like the color variable is declared correctly

Comment (taifunhaeberle) [inline src/less/components/_forms-help.less]: That's fine, see http://caniuse.com/#feat=currentcolor

**Comment by HybrisRawls** [comment]
in this case do we even need to define the color since it would inherit it?

File: src/less/components/_forms-validationandhelp.less (hunk 1 above)
Comment (saadmhybris) [inline src/less/components/_forms-validationandhelp.less]: done.

**Pull request merged**